### PR TITLE
IT WORKSSS

### PR DESCRIPTION
**Pull request by vtantra**
IT WORKSSSSS

### CODE REVIEW

File: client/testapp/api.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  return client.query({
+    query: gql`
+      {
+        helloWorld
+      }
+    `
+  });
+}
+
+export function fetchArtist(artistId: number) {
```
Comment (julialuo) [inline client/testapp/api.ts]: Annotate return types too please :)

File: client/testapp/detailsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
+import { NavigationState, NavigationScreenProp } from "react-navigation";
+import ApolloClient from "apollo-boost";
+import { ApolloProvider } from "@apollo/react-hooks";
+import { fetchHello, fetchArtist } from "./api";
+
+let nAtsts: number = 2;
```
Comment (julialuo) [inline client/testapp/detailsScreen.tsx]: Make variable names more readable

File: client/testapp/detailsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    return (
+      <View style={styles.container}>
+        <Text>Here's a random artist you should listen to!</Text>
+        <Text>{this.state.artist}</Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (julialuo) [inline client/testapp/detailsScreen.tsx]: It is good practice to move this stylesheet into a separate file (under a separate `styles` directory), and then import it here

File: client/testapp/homeScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      this.state.newArtistName,
+      this.state.newArtistAge
+    );
+    this.closeArtistModal();
+  };
+
+  render() {
+    const { navigate } = this.props.navigation;
+    return (
+      <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
```
Comment (julialuo) [inline client/testapp/homeScreen.tsx]: Try to avoid inline styling

**Comment by ekwkk** [comment]
In the future, let's try to make PRs short like less than 200 lines. :) But it's ok this time because of the configs

**Pull request merged**